tests/integration/test_jbod_balancer/test.py

```diff
@@ -67,7 +67,7 @@ def check_balance(node, table):
                 partition, disk_name
         )
         GROUP BY partition
-        HAVING gini < 0.1
+        HAVING gini < 0.2
```

Comment:
It kind of worries me about flakiness that we do `level > 0` but rely on calling time.sleep(1) after the inserts to do the merges. Could the flakiness comes from not enough parts being merged in one of the disks? I think it would make more sense to either check all active parts (independently of their level) or force a complete merge (`optimize final`).

Reply:
> I think it would make more sense to either check all active parts (independently of their level)

The test was designed in a way that there will be at most level-1 parts. The JBOD balancer only takes action during merge/fetch instead of insertion. That's why `level > 0` was used.

I also think it's better to wait for all merges are settled down. However, `optimize final` won't help as it only triggers one merge. I'll investigate a proper way to do that instead of `sleep(1)`.